client/web/src/repo/RepoRevisionContainer.tsx

```diff
@@ -37,9 +36,11 @@ import { RouteDescriptor } from '../util/contributions'
 import { CopyPathAction } from './actions/CopyPathAction'
 import { GoToPermalinkAction } from './actions/GoToPermalinkAction'
 import { ResolvedRevision } from './backend'
+import { RepoRevisionChevronDownIcon, RepoRevisionMain } from './components/RepoRevision'
```

Comment:
Let's update the name of the wrapper component

```suggestion
import { RepoRevisionChevronDownIcon, RepoRevisionWrapper } from './components/RepoRevision'
```